src/pages/PlanPage/subRoutes/CurrentOrgPlan/BillingDetails/Address/AddressForm.tsx

```diff
@@ -22,15 +23,49 @@ function AddressForm({
   owner,
 }: AddressFormProps) {
   const elements = useElements()
+  useEffect(() => {
+    const updateAppearance = () => {
+      const isDarkMode = document.body.classList.contains('dark')
```

Comment:
Not sure if this is included in your above comment @ajay-sentry, thoughts on moving to derive this value from the theme context over the body? That should also protect us in the case we add more themes or change the name of this theme and accidentally missing this string in the update

Reply:
Yep yep we're on the same page here :)

Reply:
Updated!